apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -334,6 +337,17 @@ WHERE created_at BETWEEN toStartOfDay(yesterday()) AND toStartOfDay(today())
             ;
             """;
 
+    private static final String TRACE_DAILY_BI_INFORMATION = """
+                SELECT
+                     workspace_id,
+                     created_by AS user,
+                     COUNT(DISTINCT id) AS trace_count
+                FROM traces
+                WHERE created_at BETWEEN toStartOfDay(yesterday()) AND toStartOfDay(today())
```

Comment:
Again, and this table tends to have millions of rows

Reply:
replied above